### PR TITLE
update deps, hide ci/cd to examples, update toasts, rebuild routes-ma…

### PR DESCRIPTION
**Pull request by lukachi**
…in-app structure

### CODE REVIEW

File: src/layouts/MainLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { HTMLAttributes } from 'react'
+import { Outlet } from 'react-router-dom'
+
+import { AppNavbar } from '@/common'
+
+export default function MainLayout({
+  ...rest
+}: HTMLAttributes<HTMLDivElement>) {
+  return (
+    <div {...rest} className='main-layout'>
```
Comment (ArtSuslov) [inline src/layouts/MainLayout.tsx:11]: classname will be overridden by spread

File: vite.config.ts [hunk 2]
```diff
@@ -22,12 +22,10 @@ const root = path.resolve(__dirname, resolveApp('src'))
...
 
-  const isProduction = env.VITE_ENVIRONMENT === 'production'
-  const isDevelopment = env.VITE_ENVIRONMENT === 'development'
+  // const isProduction = env.VITE_ENVIRONMENT === 'production'
+  // const isDevelopment = env.VITE_ENVIRONMENT === 'development'
   const isAnalyze = env.VITE_ENVIRONMENT === 'analyze'
-  const buildVersion = env.VITE_APP_BUILD_VERSION
-
-  console.log('isAnalyze', isAnalyze)
+  // const buildVersion = env.VITE_APP_BUILD_VERSION
```
Comment (ArtSuslov) [inline vite.config.ts:28]: ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,73 +4,14 @@ All notable changes to this project will be documented in this file.
...
-
-## [1.0.0-rc.1] - 2022-09-26
-#### Added
-- Eslint
-- Stylelint
-- prettier
-- Vite bundler
-- react-router-dom
-- store
-- axios api
```
Comment (napalmpapalam) [inline CHANGELOG.md:31]: 1. `###` not `####` 
2. why u have added, changed, fixed empty under the release ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,73 +4,14 @@ All notable changes to this project will be documented in this file.
...
-## [1.0.0-rc.0] - 2022-06-30
-#### Under the hood changes
-- Initiated and setup project
+#### Fixed
 
 [Unreleased]: https://gitlab.com/distributed_lab/frontend/react-template/compare/1.0.0...main
-[1.0.0]: https://gitlab.com/distributed_lab/frontend/react-template/compare/1.0.0-rc.1...1.0.0
-[1.0.0-rc.1]: https://gitlab.com/distributed_lab/frontend/react-template/compare/1.0.0-rc.0...1.0.0-rc.1
-[1.0.0-rc.0]: https://gitlab.com/distributed_lab/frontend/react-template/tags/1.0.0-rc.0
+[1.0.0]: https://gitlab.com/distributed_lab/frontend/react-template/tags/1.0.0
```
Comment (napalmpapalam) [inline CHANGELOG.md:17]: gitlab?

File: src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { FC, HTMLAttributes, isValidElement, ReactNode } from 'react'
+
+import { Icon } from '@/common'
+import { ToastPayload } from '@/contexts/ToastsManager'
```
Comment (napalmpapalam) [inline src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx:4]: `from '@/contexts`

File: src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  payload: ToastPayload | ReactNode
+}
```
Comment (napalmpapalam) [inline src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx:8]: use type plz

File: src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+const DefaultToast: FC<Props> = ({ payload, ...rest }) => {
+  if (isValidElement(payload)) {
+    return (
+      <div {...rest} className='toast__body'>
+        {payload}
+      </div>
+    )
+  } else {
```
Comment (napalmpapalam) [inline src/contexts/ToastsManager/components/DefaultToast/DefaultToast.tsx:17]: else not needed

File: src/helpers/event-bus.ts [hunk 8]
```diff
@@ -1,17 +1,17 @@
 import { EventEmitter } from '@distributedlab/tools'
 
-export enum BUS_EVENTS {
+export enum BusEvents {
   error = 'error',
   warning = 'warning',
   success = 'success',
   info = 'info',
```
Comment (napalmpapalam) [inline src/helpers/event-bus.ts:7]: PascalCase

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (lukachi) [reply CHANGELOG.md:31]: because it's a template

Comment (napalmpapalam) [reply CHANGELOG.md:31]: remove added, changed, fixed then, leave just version

**Pull request merged**